src/auth/server_impl/mod.rs

```diff
@@ -97,6 +97,14 @@ pub async fn try_extract_identity(
     fetch_identity_from_kv(kv, principal).await
 }
 
+
+pub async fn webhook_kv_update_impl(uid: String) -> Result<(),ServerFnError>{
+    let kv: KVStoreImpl = expect_context();
+
+    Ok(kv.write(uid, false.to_string()).await?)
```

Comment:
KVStore is technically only used for authentication.
Is this absolutely necessary to do in redis? I think using analytics for this makes more sense

Reply:
Different kv store (different instance) is proposed.. after discussion with Saikat